### PR TITLE
Features/all day

### PR DESCRIPTION
**Pull request by rkorzeniec**
This PR makes the `All day` section of the calendar irrelevant, as it takes the all day events and dynamically splits them into full day events.

before:
Screenshot 2019-08-23 at 21 11 54

after:
Screenshot 2019-08-23 at 21 12 06

### CODE REVIEW

File: src/api/events-collection.js [hunk 1]
```diff
@@ -31,6 +39,19 @@ export default class EventsCollection {
         return event;
     }
 
+    prepareEventAttributes(eventAttrs, eventDay) {
+        if (eventDay === undefined) {
```
Comment (mmarinovic) [inline src/api/events-collection.js]: Is `null` ok?

Comment (rkorzeniec) [reply src/api/events-collection.js]: I will check

File: src/api/events-collection.js [hunk 2]
```diff
@@ -31,6 +39,19 @@ export default class EventsCollection {
...
+            return eventAttrs;
+        }
+
+        const attrs = Object.assign({}, eventAttrs);
```
Comment (Neophy7e) [inline src/api/events-collection.js]: As this code is es6 you could do `{... eventAttrs}` to . have the same result same for the return value

Comment (rkorzeniec) [reply src/api/events-collection.js]: oh! perfect, forgot about the new merge syntax

File: src/api/events-collection.js (hunk 1 above)
Comment (rkorzeniec) [reply src/api/events-collection.js]: @mmarinovic thanks for the input, unfortunately not
before:
Screenshot 2019-08-26 at 17 01 48


after:
Screenshot 2019-08-26 at 17 00 54

**Comment by rkorzeniec** [comment]
@Neophy7e @mmarinovic I asked the owner of this component whether he would accept this kind of change https://github.com/nathanstitt/dayz/issues/25

**Comment by Neophy7e** [comment]
Nice ! Just saw that he answered you https://github.com/nathanstitt/dayz/issues/25 actually he's not.keavong his lib dead 🙌

**Comment by rkorzeniec** [comment]
@mmarinovic @Neophy7e I made the change backwards compatible, will also open the PR on the original component

File: src/api/events-collection.js (hunk 1 above)
Comment (mmarinovic) [reply src/api/events-collection.js]: Sorry, I meant if passing null is ok?
What about just doing  `if (!eventDay)`

**Comment by rkorzeniec** [comment]
@Neophy7e @mmarinovic I opened same PR also on the original component, let's see if the owner likes it, if yes, I will close our repo, if not I will merge this PR

**Comment by Neophy7e** [comment]
@rkorzeniec sounds good

**Comment by Neophy7e** [comment]
@rkorzeniec should we remove this repo ?

**Comment by rkorzeniec** [comment]
yes I think so

**Pull request closed without merging**